core/domain/exp_jobs_one_off_test.py

```diff
@@ -2178,7 +2178,7 @@ def test_no_action_is_performed_for_deleted_exploration(self):
             self, exp_jobs_one_off.TranslatorToVoiceArtistOneOffJob)
 
 
-class TestDeleteStateIdMappingModelsOneOffJob(test_utils.GenericTestBase):
+class DeleteStateIdMappingModelsOneOffJobTests(test_utils.GenericTestBase):
```

Comment:
I think there are other files as well which follow this convention. Can you do a grep for `class Test`?

Reply:
Yes, I checked that already but they are mock classes or base classes and are not being used in running the test.